README.md

```diff
@@ -13,7 +13,8 @@ Use the file extension of your desired file type in the URL (`.png`, `.jpg`, or
 parameter | default | unit | purpose
 --- | --- | --- | ---
 `values` | | | (required) comma-separated list of values to graph
-`background_color` | `FFFFFF` | hex | background color of the image
+`background_color` | `FFFFFF` | hex | background color 
+`background_color` | `transparent` | text | background is transparent
```

Comment:
Hmm...thoughts about combining these into a single row?

Also, we should give more guidance about the full range of options for the colors. How about we say `color` under the `unit` column for the `*_color` parameters, then add a note beneath the table with more explanation. Something to the effect of:

> The `color` value can be anything from [this list](https://rmagick.github.io/imusage.html#color_names). You can leave out the `#` prefix for hexidecimal colors, but it will need to be encoded as `%23` for the other RGB formats.

I guess that second sentence could be a bit confusing though...maybe we just make it more consistent with:

> The `color` value can be anything from [this list](https://rmagick.github.io/imusage.html#color_names). Use `%23` instead of the `#` for the RGB formats.

and update the `default`s to match. Thoughts?